spec/ember-cli/manifest_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "ember-cli/manifest"
+
+describe EmberCli::Manifest do
+  describe "#fixup!" do
+    it "fixes paths" do
+      input = build_manifest("ember", {
+        "assets" => {
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.